packages/components/src/switch/src/SwitchField.tsx

```diff
@@ -86,15 +97,11 @@ function SwitchField(props: SwitchFieldProps, ref: ForwardedRef<HTMLDivElement>)
             ]}
         >
             <div
-                {...otherProps}
+                {...mergeProps(otherProps, renderProps)}
                 ref={ref}
-                className={classNames}
-                style={mergedStyles}
                 slot={slot ?? undefined}
                 data-disabled={isDisabled}
-            >
-                {children}
```

Comment:
I think RAC usually does renderProps.children here, but because that's all that's here, I think it works fine since it's added when you put renderProps

Reply:
Yeah I saw that, but this technically works too, don't mind doing any way.